packages/chain-adapters/src/utxo/UtxoBaseAdapter.ts

```diff
@@ -132,6 +133,25 @@ export abstract class UtxoBaseAdapter<T extends UtxoChainId> implements IChainAd
     return { ...this.defaultBIP44Params, ...params }
   }
 
+  getBIP44Params({ accountNumber, accountType }: GetBIP44ParamsInput): BIP44Params {
+    if (accountNumber < 0) {
+      throw new Error('accountNumber must be >= 0')
+    }
+    const purpose: number = (() => {
```

Comment:
nitpick use inferred type
```suggestion
    const purpose = (() => {
```